### ISSUE TITLE
AttributeError: 'Tag' object has no attribute 'value'

### ISSUE THREAD

**Issue by fa16-030** [pyannote/pyannote-audio#1387]
```
from pyannote.audio import Pipeline
pipeline = Pipeline.from_pretrained("pyannote/speaker-diarization@2.1", use_auth_token="my_token")
```
After running above command of loading the model on colab. Its give error.

```
[/usr/local/lib/python3.10/dist-packages/hyperpyyaml/core.py](https://localhost:8080/#) in _walk_tree_and_resolve(key, current_node, tree, file_path)
    374     # Base case, handle tags
    375     if hasattr(current_node, "tag"):
--> 376         tag_value = current_node.tag.value or ""
    377 
    378         # Placeholders should have been replaced before now

AttributeError: 'Tag' object has no attribute 'value'

```

Is this some version issue ? because colab is updating continuously. One week before, Its working fine!

**Comment by marco-ramirezpp** [comment]
The problem is a dependency of the library (ruamel.yaml)
To fix, downgrade to 0.17.28 as follows:

pip install ruamel.yaml==0.17.28

**Comment by fa16-030** [comment]
Thanks, it worked !
